src/app/accounts/update-account-withdraw-fee.ts

```diff
@@ -0,0 +1,22 @@
+import { checkedToWithdrawFee } from "@domain/accounts"
+import { AccountsRepository } from "@services/mongoose"
+
+export const updateAccountWithdrawFee = async ({
+  id,
+  fee,
+}: {
+  id: AccountId
+  fee: number
+}): Promise<Account | ApplicationError> => {
+  const checkedFee = checkedToWithdrawFee(fee)
+  if (checkedFee instanceof Error) return checkedFee
+
+  const accountsRepo = AccountsRepository()
```

Comment:
```
const account = await AccountsRepository().findById(id)
```
=> can delete 2 lines.

Reply:
We're using accountsRepo in two places. One is here, and one is a few lines below for updating the account.
If we use `await AccountsRepository().findById(id)` then the update call with have to be changed to `return AccountsRepository().update(account)` which would mean two initializations of `AccountsRepository`.

Is this what you're suggesting?

Reply:
No I missed that its being used twice.